### PR TITLE
Vehicle information improvements

### PR DESCRIPTION
**Pull request by jncn**
- Added platform per stop
- Added station id to `stop`, even if `fast=true`

### CODE REVIEW

File: api/APICall.php [hunk 1]
```diff
@@ -83,8 +83,8 @@ public function executeCall()
         try {
             $this->dataRoot->fetchData($this->request, $this->request->getSystem());
             $this->dataRoot->printAll();
-            $this->writeLog();
         } catch (Exception $e) {
+            $this->writeLog($e);
```
Comment (pietercolpaert) [inline api/APICall.php]: This doesn't seem correct... Why did you do this?

**Comment by pietercolpaert** [comment]
I can pull in style fixes from style-ci afterwards... Will merge even with style problems.

One comment needs to be addressed though

**Comment by pietercolpaert** [comment]
Can you "view source" on that file and see what the "extra content" is at the end of the xml?

**Comment by jncn** [comment]
Sorry, that was more useful indeed:
```
<b>Warning</b>:  Missing argument 1 for APICall::writeLog(), called in /Users/Jan/Sites/iRail-fork/api/APICall.php on line 86 and defined in <b>[path to]/api/APICall.php</b> on line <b>147</b><br />
```

**Comment by pietercolpaert** [comment]
Aha! Will fix that myself. Can you revert https://github.com/jncn/iRail/commit/12d7f4742c799446482cc5ea676e041b27416bbd from this PR?

**Comment by pietercolpaert** [comment]
Thanks! Will now test manually in development, apply style fixes and put in production!

**Comment by jncn** [comment]
You're welcome, I'm trying to figure out what went wrong to the styles...

**Comment by pietercolpaert** [comment]
No worries, just some whitespace nitpicking of the styleci tool. It can apply the fixes automatically, so I usually don't bother too much :)

**Pull request merged**